bin/setup.sh

```diff
@@ -16,14 +16,19 @@ git clone https://github.com/viamrobotics/viam-cpp-sdk.git
 pushd viam-cpp-sdk
 
 # NOTE: If you change this version, also change it in the `conanfile.py` requirements
-git checkout releases/v0.8.0
+git checkout releases/v0.9.0
```

Comment:
Per the comment, also update conanfile.py.

Reply:
Hah. Thanks.